applications/tasks/src/modules/kt_users.erl

```diff
@@ -0,0 +1,556 @@
+    kz_json:set_value([Category, Action], kz_json:from_map(action(Action)), JObj).
+
+%%%=============================================================================
+%%% Actions
+%%%=============================================================================
+
+%%------------------------------------------------------------------------------
+%% @doc
+%% @end
+%%------------------------------------------------------------------------------
+-spec action(kz_term:ne_binary()) -> map().
+action(<<"import">>) ->
+    #{<<"description">> => <<"Bulk-import of users">>
+     ,<<"doc">> => <<"Bulk create a user for the defined account id.\n"
+                     "For each user created, return fields:\n"
+                     "* `id`: doc id it is assigned to (32 alphanumeric characters).\n"
+                     "* `username`: The users username (Unique)\n"
+                     "* `first_name`: The users first_name\n"
+                     "* `last_name`: The users last_name\n"
+                     "* `account_id`: account it is assigned to (32 alphanumeric characters).\n"
+                     "Note: account creating the task (or `auth_by` account) must have permissions on the affecting account.\n"
+                   >>
+     ,<<"expected_content">> => <<"text/csv">>
+     ,<<"mandatory">> => mandatory_import_fields()
+     ,<<"optional">> => optional_import_fields()
+     ,<<"return_headers">> => output_header() ++ [?OUTPUT_CSV_HEADER_ERROR]
+     };
+
+action(<<"delete">>) ->
+    #{<<"description">> => <<"Bulk-remove users">>
+     ,<<"doc">> => <<"Forces users to be deleted from an account\n"
+                     "Note: account creating the task (or `auth_by` account) must have permissions on the affecting account.\n"
+                   >>
+     ,<<"expected_content">> => <<"text/csv">>
+     ,<<"mandatory">> => mandatory_delete_fields()
+     ,<<"optional">> => optional_delete_fields()
+     ,<<"return_headers">> => output_header() ++ [?OUTPUT_CSV_HEADER_ERROR]
+     }.
+
+%%%=============================================================================
+%%% Verifiers
+%%% These are called during CSV first upload on all rows to verify the CSV data
+%%%=============================================================================
+
+%%------------------------------------------------------------------------------
+%% @doc
+%% @end
+%%------------------------------------------------------------------------------
+-spec verify_account_id(kz_term:ne_binary()) -> boolean().
+verify_account_id(?MATCH_ACCOUNT_RAW(_)) -> 'true';
+verify_account_id(_) -> 'false'.
+
+-spec verify_user_id(kz_term:ne_binary()) -> boolean().
+verify_user_id(?MATCH_USER_ID(_)) -> 'true';
+verify_user_id(_) -> 'false'.
+
+-spec verify_email(kz_term:ne_binary()) -> boolean().
+verify_email(Cell) ->
+    EmailRegExp = "^[-!#$%&'*+\\/0-9=?A-Z^_a-z{|}~](\\.?[-!#$%&'*+\\/0-9=?A-Z^_a-z`{|}~])*@[a-zA-Z0-9](-*\\.?[a-zA-Z0-9])*\\.[a-zA-Z](-?[a-zA-Z0-9])+$",
```

Comment:
`case binary:split(Cell, <<"@">>) of [_User, _Domain] -> 'true'; _ -> 'false' end` not good enough? :)